resources/public/js/static-js.js

```diff
@@ -20,24 +20,10 @@ function OCStaticMailchimpApiSubmit(e, form, success, fail){
 }
 
 function OCStaticGetCookie(name) {
```

Comment:
@bago2k4 can you leave some comments on how this function is used? 

My concern here is how much the code has changed and the fact that decodeURI is no longer used. Anyway something to clear up the functions use will be good.

Reply:
Good point about decodeURI not used, i'll add it.
For the missing comment i thought the name was pretty self explanatory: it's used to get a single cookie value given its name.
I'll add the comment too though